homeassistant/components/google_travel_time/config_flow.py

```diff
@@ -106,7 +109,11 @@
                 translation_key=CONF_TIME_TYPE,
             )
         ),
-        vol.Optional(CONF_TIME, default=""): cv.string,
+        vol.Optional(CONF_TIME): TextSelector(
+            TextSelectorConfig(
+                type=TextSelectorType.TIME,
+            )
+        ),
```

Comment:
What's the difference between a TextSelector and a TimeSelector? (as in, I don't have the text selector with type=time top of mind, so I am interested why you chose this one)

Reply:
A text selector lets one input arbitrary text, e.g. `now` is explicitly supported. Whereas the new SDK only supports real timestamps. I am using the TextSelectorType.TIME to be sure that the config contains values which I can safely pass to `dt_util.parse_time()`

Reply:
Is there an argument against the TimeSelector? As that would directly get you a time object. And if I understand correctly we never have to input arbitrary texts again so then that would be fine right?

Reply:
Yep. Gonna change it

Reply:
Done